docker/docker-compose.1604.52.yaml

```diff
@@ -20,35 +20,35 @@ services:
     environment:
       - MAX_ALLOCS_ALLOWED_1000_addHandlers=47050
       - MAX_ALLOCS_ALLOWED_1000_addHandlers_sync=40050
-      - MAX_ALLOCS_ALLOWED_1000_addRemoveHandlers_handlercontext=9035
-      - MAX_ALLOCS_ALLOWED_1000_addRemoveHandlers_handlername=9035
-      - MAX_ALLOCS_ALLOWED_1000_addRemoveHandlers_handlertype=9035
+      - MAX_ALLOCS_ALLOWED_1000_addRemoveHandlers_handlercontext=9050
+      - MAX_ALLOCS_ALLOWED_1000_addRemoveHandlers_handlername=9050
+      - MAX_ALLOCS_ALLOWED_1000_addRemoveHandlers_handlertype=9050
       - MAX_ALLOCS_ALLOWED_1000_autoReadGetAndSet=32050
-      - MAX_ALLOCS_ALLOWED_1000_autoReadGetAndSet_sync=50
+      - MAX_ALLOCS_ALLOWED_1000_autoReadGetAndSet_sync=0
       - MAX_ALLOCS_ALLOWED_1000_getHandlers=12050
-      - MAX_ALLOCS_ALLOWED_1000_getHandlers_sync=50
-      - MAX_ALLOCS_ALLOWED_1000_reqs_1_conn=30540
-      - MAX_ALLOCS_ALLOWED_1000_tcpbootstraps=4100
-      - MAX_ALLOCS_ALLOWED_1000_tcpconnections=178010
+      - MAX_ALLOCS_ALLOWED_1000_getHandlers_sync=37
+      - MAX_ALLOCS_ALLOWED_1000_reqs_1_conn=30450
+      - MAX_ALLOCS_ALLOWED_1000_tcpbootstraps=4050
+      - MAX_ALLOCS_ALLOWED_1000_tcpconnections=178050
       - MAX_ALLOCS_ALLOWED_1000_udp_reqs=16050
-      - MAX_ALLOCS_ALLOWED_1000_udpbootstraps=2000
+      - MAX_ALLOCS_ALLOWED_1000_udpbootstraps=2050
       - MAX_ALLOCS_ALLOWED_1000_udpconnections=102050
       - MAX_ALLOCS_ALLOWED_1_reqs_1000_conn=470050
-      - MAX_ALLOCS_ALLOWED_bytebuffer_lots_of_rw=2100
-      - MAX_ALLOCS_ALLOWED_creating_10000_headers=100 # 5.2 improvement 10000
-      - MAX_ALLOCS_ALLOWED_decode_1000_ws_frames=2000
-      - MAX_ALLOCS_ALLOWED_encode_1000_ws_frames_holding_buffer=1000
-      - MAX_ALLOCS_ALLOWED_encode_1000_ws_frames_holding_buffer_with_space=1000
-      - MAX_ALLOCS_ALLOWED_encode_1000_ws_frames_new_buffer=5010
-      - MAX_ALLOCS_ALLOWED_encode_1000_ws_frames_new_buffer_with_space=5010
-      - MAX_ALLOCS_ALLOWED_future_lots_of_callbacks=75010
-      - MAX_ALLOCS_ALLOWED_modifying_1000_circular_buffer_elements=50
-      - MAX_ALLOCS_ALLOWED_modifying_byte_buffer_view=2010 # 5.2 improvement 4000
-      - MAX_ALLOCS_ALLOWED_ping_pong_1000_reqs_1_conn=4440
+      - MAX_ALLOCS_ALLOWED_bytebuffer_lots_of_rw=2050
+      - MAX_ALLOCS_ALLOWED_creating_10000_headers=0
+      - MAX_ALLOCS_ALLOWED_decode_1000_ws_frames=2050
+      - MAX_ALLOCS_ALLOWED_encode_1000_ws_frames_holding_buffer=3
+      - MAX_ALLOCS_ALLOWED_encode_1000_ws_frames_holding_buffer_with_space=3
+      - MAX_ALLOCS_ALLOWED_encode_1000_ws_frames_new_buffer=5050
+      - MAX_ALLOCS_ALLOWED_encode_1000_ws_frames_new_buffer_with_space=5050
+      - MAX_ALLOCS_ALLOWED_future_lots_of_callbacks=75002
```

Comment:
How does this end up with a trailing `2`?

Reply:
that is an excellent question!